### PR TITLE
ci: fix commands/general specs

### PR DESCRIPTION
**Pull request by rerorero**
A part of https://github.com/appium/appium/issues/16402

This PR enables specs under `test/functional/commands/general` directory on CI

### CODE REVIEW

**Comment by rerorero** [comment]
I think it's ready for review now. Please take a look.

File: test/functional/desired.js [hunk 1]
```diff
@@ -63,6 +63,7 @@ const GPS_DEMO_CAPS = amendCapabilities(GENERIC_CAPS, {
 
 const BROWSER_CAPS = amendCapabilities(GENERIC_CAPS, {
   browserName: 'Chrome',
+  'appium:noReset': true,
```
Comment (KazuCocoa) [inline test/functional/desired.js:66]: Is this necessary?

Comment (rerorero) [reply test/functional/desired.js]: I think no, I don't remember how I did it... I removed it 👍

Comment (rerorero) [reply test/functional/desired.js]: https://github.com/appium/appium-uiautomator2-driver/pull/583/commits/663067b4ebacd127ec1c86509f7bb65aa6c1fd79

**Review by KazuCocoa — APPROVED** [review]
lgtm, thanks!

**Pull request merged**